overlord/snapstate/export_test.go

```diff
@@ -269,8 +269,12 @@ var (
 	MissingDisabledServices = missingDisabledServices
 )
 
-func (m *SnapManager) MaybeUndoRemodelBootChanges(t *state.Task) error {
-	return m.maybeUndoRemodelBootChanges(t)
+func (m *SnapManager) MaybeUndoRemodelBootChanges(t *state.Task) (restartRequested, rebootRequired bool, err error) {
+	maybeRst, err := m.maybeUndoRemodelBootChanges(t)
```

Comment:
I read this as "maybe reset", which is similar but not 100% equivalent. I'd call this variable `restartInfo`.